frontend/src/components/Profile/Profile.tsx

```diff
@@ -37,7 +36,29 @@ const Profile = () => {
     return <DefaultPage className="aha__profile">{view}</DefaultPage>;
 };
 
-export const ProfileView = (data) => {
+interface ProfileViewProps {
+    messages: {
+        title: string;
+        summary: string;
+        continue: string;
+        points: string;
+    };
+    scores: {
+        // This might need to be renamed to block_slug and experiment_slug
+        // after the great refactor of Summer 2024
+        experiment_slug: string;
+        experiment_name: string;
```

Comment:
Check if this shouldn't be already be renamed before merging.

Reply:
Good point. This is exactly why I placed the comment :-)